doc/open_datasets.rst

```diff
@@ -0,0 +1,12 @@
+Open datasets
+=============
+
```

Comment:
I also have some open datasets which I've uploaded to Zenodo, which might be of interest:

- 4D-STEM atomic resolution dataset of a perovskite oxide thin film: https://zenodo.org/record/4727847/files/011_big_film_512x512_updated.hspy?download=1 (only this dataset will load easily in HyperSpy/pyxem)
- Magnetic STEM-DPC dataset: https://zenodo.org/record/4312960